addons/point_of_sale/models/account_move.py

```diff
@@ -71,3 +71,8 @@ def _stock_account_get_anglo_saxon_price_unit(self):
         if sudo_order:
             price_unit = sudo_order._get_pos_anglo_saxon_price_unit(self.product_id, self.move_id.partner_id.id, self.quantity)
         return price_unit
+
+    def _check_edi_line_tax_required(self):
+        if self.product_id.type == 'combo':
```

Comment:
Quick note: this logic will have to be moved to the `account` module in the forward-port for 18 (because the combo logic was moved to the `product` module).